src/SFA.DAS.AdminService.Web/Infrastructure/CompaniesHouseApiClient.cs

```diff
@@ -22,7 +22,7 @@ public class CompaniesHouseApiClient:ICompaniesHouseApiClient
             _logger = logger;
 
             _httpClient = new HttpClient { BaseAddress = new Uri(baseUri) };
-            _httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", _tokenService.GetToken());
+            _httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", _tokenService.GetToken().GetAwaiter().GetResult());
```

Comment:
This is the quickest way to make it work with the changes to `IRoatpTokenService`.

Reply:
This is hair-raising - so the presumably the CompaniesHouseApiClient is not used because how could it be used when it's using a token generated to talk to the Roatp V2 Apply service??

Reply:
I've raised a backlog item to investigate this: https://skillsfundingagency.atlassian.net/browse/QF-1046